payments-core/src/main/java/com/stripe/android/payments/bankaccount/CollectBankAccountLauncher.kt

```diff
@@ -84,7 +86,8 @@ interface CollectBankAccountLauncher {
                 hostedSurface = null,
                 hostActivityLauncher = activity.registerForActivityResult(CollectBankAccountContract()) {
                     callback(it.toUSBankAccountResult())
-                }
+                },
+                financialConnectionsAvailability = GetFinancialConnectionsAvailability(elementsSession = null),
```

Comment:
Should we assert that this is not `null` here? It’ll at least be `Lite`, right?

Reply:
yeah, but we'll be adding a check in the activity directly, so that we can properly fail with `CollectBankAccountResult.Failed` (rather than crashing here)

Reply:
Got it! Just want to make sure we don’t carry the nullability around needlessly.